aten/src/ATen/native/native_functions.yaml

```diff
@@ -2599,6 +2599,14 @@
 
 - func: sum.DimnameList_out(Tensor self, Dimname[1] dim, bool keepdim=False, *, ScalarType? dtype=None, Tensor(a!) out) -> Tensor(a!)
 
+- func: nansum(Tensor self, *, ScalarType? dtype=None) -> Tensor
+  variants: function, method
```

Comment:
I think you want to add `use_c10_dispatcher: full` here and for the other functions.

Reply:
Thanks. Done.